### PR TITLE
Replacing slug to slugger

### PR DESCRIPTION
**Pull request by umbobabo**

### CODE REVIEW

File: omniture-config.es6 [hunk 1]
```diff
@@ -1,24 +1,13 @@
...
-  multicharmap: slug.multicharmap,
-};
+import slugger from 'slugger';
 import OmnitureUtils from '@economist/react-i13n-omniture/OmnitureUtils';
+function slug(string) {
+  let sluggedString = '';
+  if (typeof string === 'string') {
+    sluggedString = slugger(string, { replacement: '_' });
+  }
+  return sluggedString;
```
Comment (keithamus) [inline omniture-config.es6]:
```js
function slug(string) { return slugger(String(string || ''), { replacement: '_' }); }
```
would be be better here.

Comment (umbobabo) [inline omniture-config.es6]: The idea was to prevent slugger to fail if the string is empty, that is different from this implementation, but I go for that.

**Comment by umbobabo** [comment]
Do not merge, I'm releasing a depend PR.

File: index.css [hunk 2]
```diff
@@ -66,4 +66,6 @@
 .ad-panel__container {
     width: 100%;
     box-sizing: border-box;
+    background-color: #f0f0f0;
+    padding: 3%;
```
Comment (keithamus) [inline index.css]: this change seems unrelated

Comment (umbobabo) [inline index.css]: It is but they are asking me to fix the ad quickly. So...
